README.md

```diff
@@ -1,13 +1,15 @@
 # Cluedo - Mystery Randomizer
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project is to make a Cluedo random mysterypicker. The goal is to practice using arrays, objects and loops.
 
 ## The problem
 
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
+I started with coding the javascript part, and making all the functions on the page work. After that I started with the styling of the page. This project I did not use Trello for planning as much as in the earlier projects, and to next assignment I will go back to my planning-board in Trello.
```

Comment:
Great idea to start ideation in Trello. Agree with re-usable functions. It´s something that takes a bit of time but valuable for readability of code for oneself and others :)